packages/select/src/Select/components/FloatingSelectWrapper/FloatingSelectWrapper.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable jsx-a11y/no-autofocus */
+import React from "react"
+import { FocusScope } from "@react-aria/focus"
+import { useOverlay, DismissButton } from "@react-aria/overlays"
+import { mergeProps } from "@react-aria/utils"
+import { useMenuTriggerContext } from "../../provider"
+import styles from "./FloatingSelectWrapper.module.scss"
+export interface FloatingSelectWrapperProps {
+  children: React.ReactNode
+}
+
+export const FloatingSelectWrapper: React.VFC<FloatingSelectWrapperProps> = ({
```

Comment:
if you use React.FC instead of VFC, you wont need FloatingSelectWrapperProps as FC comes with the children prop

Reply:
We haven't changed to `FC` yet because of backwards compatibility. The old `FC` includes `children` which we don't want. When we no longer support the old version (I think 16), we can port it over.

Reply:
@HeartSquared could we not use the React type ? then there is no need to change it later ?
```suggestion
export const FloatingSelectWrapper = ({ children }: FloatingSelectWrapperProps) => {
```

Reply:
Technically yes, however you'll also need to put in the return type of `JSX.Element` (which `FC/VFC` do for you). Currently it allows you to infer the return type, but I do want to make our linting stricter in the future. Typing the function removes the need to put in the return type (since it's part of the function signature).

I'm not actually opposed to either method as long as we are consistent. Currently the `VFC` pattern is what we've opted for, so we should keep to it until we revise our patterns.